modules/claims_api/spec/fixtures/form_526_json_api.json

```diff
@@ -50,7 +50,7 @@
       "standardClaim": false,
       "autoCestPDFGenerationDisabled": true,
       "claimDate": "1990-01-03",
-      "applicationExpirationDate": "2015-08-28T19:53:45+00:00",
+      "applicationExpirationDate": "2300-08-28T19:53:45+00:00",
```

Comment:
updated to pass validation requirement that the expiration date is in the future

Reply:
can we use Timecop here instead?

Reply:
I assume you mean within the request spec? updated to that 👍